Tests/conf.json

```diff
@@ -2615,7 +2615,11 @@
             "playbookID": "DBotCreatePhishingClassifierV2FromFile-Test",
             "timeout" : 60000,
             "fromversion": "4.5.0"
-        }
+        },
+	{
```

Comment:
please make sure the curly brackets are linted correctly and add the integration id here as well